napari_cellseg3d/dev_scripts/correct_labels.py

```diff
@@ -0,0 +1,369 @@
+import threading
+import time
+import warnings
+from functools import partial
+from pathlib import Path
+
+import napari
+import numpy as np
+import scipy.ndimage as ndimage
+from napari.qt.threading import thread_worker
+from tifffile import imread, imwrite
+from tqdm import tqdm
+
+import napari_cellseg3d.dev_scripts.artefact_labeling as make_artefact_labels
+from napari_cellseg3d.code_models.model_instance_seg import binary_watershed
+
+# import sys
+# sys.path.append(str(Path(__file__) / "../../"))
+"""
+New code by Yves Paychère
```

Comment:
Should there be a link from where this comes from?

Reply:
This is code from the MLCourse project of last semester; so it was written specifically for this :) I can link to my repo where the original code lives, though ?